docs/sections/esys.md

```diff
@@ -1,7 +1,196 @@
 # Energiesystem
 
-Hier wird der Aufbau des Energiesystem beschrieben. Die Generierung der
-Energiesystemdaten für die App ist Teil der
-[Developer Docs](../sections/esys_dev.md).
+Im Folgenden wird der Aufbau des Energiesystem von Anhalt-Bitterfeld und
+Wittenberg (ABW) beschrieben. Die Generierung der Energiesystemdaten für die
+App ist Teil der [Developer Docs](../sections/esys_dev.md).
 
-**TBD**
+## Das Energiesystem im Überblick
+
+Das in den Berechnungen von Digiplan zum Einsatz kommende Energiesystem ist
+hier abgebildet:
+
+![Energy system schema](../img/esys/DigiPlan_Energiesystem.svg)
+
+In den Modellen wird das Energiesystem von Anhalt-Bitterfeld und Wittenberg
+zum Jahr 2045 aufgestellt. Dabei wird sich am derzeitigen Energiesystem
+orientiert, welches auf den Daten des Marktstammdatenregister (MaStR) beruht.
+
+## Annahmen und Vereinfachungen
+
+Wir treffen die folgenden Annahmen und Vereinfachungen:
+
+- Für die Prognosen werden [Langfristszenarien](https://langfristszenarien.de/enertile-explorer-de/index.php)
+  verwendet.
+- Biomasse wird unterteilt in: feste, gasförmige und flüssige Biomasse.
+- Zur Abbildung der Biomasse wird vom Status-quo in ABW ausgegangen.
+- Der Status-qup ist das Jahr 2022.
+- Für die Vorhersage zur Entwicklung der Biomasse wird sich auf den Bericht der
+  DBFZ gestützt: ["Technoökonomische Analyse und Transformationspfade des
+  energetischen Biomassepotentials (TATBIO)"](https://www.bmwk.de/Redaktion/DE/Publikationen/Studien/technooekonomische-analyse-und-transformationspfade-des-energetischen-biomassepotentials.pdf?__blob=publicationFile&v=1).
+- Flüssige Biomasse wird im Jahr 2045 ausschließlich im Verkehrsektor
+  eingesetzt.
+- Verkehrssektor wird vernachlässigt. Demnach entfällt der Einsatz flüssiger
+  Biomasse.
+- Es wird kein Biogas importiert: Sämtliches Biogas wird regional erzeugt.
+- Der Bus Methan steht vereinfachend für einen Bus für klimaneutrale Gase.
+  Es handelt sich um eine Mischung aus grünem Wasserstoff und Biomethan.
+  Immer wenn von Methan gesprochen wird, ist diese Gasmischung gemeint.
+- Der Energieträger Braunkohle ist nur relevant zur Berechnungen der Emissionen
+  zum Status-quo.
+- Für das Jahr 2045 ist kein Import von Braunkohle vorhergesehen.
+- Es wird das Jahr 2045 in stündlichen Schritten betrachtet.
+- Sämtliche Zeitreihen liegen in dieser Auslösung vor oder werden in diese
+  Auflösung übersetzt.
+- Es handelt sich um eine Dispatch Optimierung.
+- Sämtliche Kapazitäten bis auf die der Importe und Exporte sowie die der
+  Speichertechnologien sind fix gesetzt.
+- Die Optimierung findet nach dem Perfect forsight Prinzip über den
+  Zeithorizont eines Jahres statt.
+- TODO: aggregation (ABW, central+decentral, ...)
```

Comment:
I was not sure about this. Partly it is already explained in the md files of the data processing itself. What else would you like to have mentioned on this point @nesnoj?